tests/template/my_model.F90

```diff
@@ -0,0 +1,64 @@
+! Copyright (C) 2022 National Center for Atmospheric Research
+! SPDX-License-Identifier: Apache-2.0
+!
+program test_my_model
+
+  use my_model,                        only : my_model_t
+
+  implicit none
+
+  call test_my_model_t( )
+
+contains
+
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+  !> Tests the my_model_t stub class
+  subroutine test_my_model_t( )
+
+    use aero_array,                      only : array_t
+    use aero_constants,                  only : rk => real_kind
+    use aero_grid,                       only : grid_t
+    use aero_model,                      only : model_t
+    use aero_state,                      only : state_t
+    use aero_util,                       only : assert, almost_equal
+
+    class(model_t), pointer :: model
+    class(state_t), pointer :: state
+    type(grid_t) :: optics_grid
+    class(array_t), pointer :: interfaces, od, od_ssa, od_asym
+    real(kind=rk), allocatable :: od_a(:), od_ssa_a(:), od_asym_a(:)
+
+    model => my_model_t( "" )
```

Comment:
is the argument a model name? Maybe we could set it to something and check that the name is correct.

Reply:
it's set up to be a path to a aerosol model-specific configuration file. I'm hoping to use it once we add NetCDF support